packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/deleteDevice/delete-device.controller.js

```diff
@@ -0,0 +1,37 @@
+export default class DeleteDeviceController {
+  /* @ngInject */
+  constructor($translate, $stateParams, softphoneService, TucToast) {
+    this.$translate = $translate;
+    this.TucToast = TucToast;
+    this.$stateParams = $stateParams;
+    this.softphoneService = softphoneService;
+  }
```

Comment:
**nitpick:** Dont inject `$stateParams` in controller but add an input to the component and send it from routing file.